api/MediaCapabilities.json

```diff
@@ -145,10 +145,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": false
```

Comment:
This is wrong `navigator.mediaCapabilities.decodingInfo` exists in Safari 13.1, and `'decodingInfo' in MediaCapabilities.prototype` is true. @vinyldarkscratch something seems to have gone wrong with the process here.

Reply:
Actually, in order to get this huge PR landed, I'll revert it. Can you check it out separately?